### DISCUSSION TITLE
Colloqui sono a distanza

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#144] [Richieste di aiuto] [answered]
Buongiorno ,sarebbe possibile implementare la prenotazione di colloqui in presenza e non solo a distanza?

**Comment by trinko** [comment, marked as answer]
E' già presente, ma non c'è un parametro per gestirla.

Al momento puoi solo intervenire nel template relativo alle richieste di colloquio e commentare le righe che riportano le informazioni sul colloquio a distanza. Per il resto la gestione non cambia.
